applications/DEMApplication/custom_conditions/dem_wall.cpp

```diff
@@ -212,9 +212,11 @@ void DEMWall::CalculateNormal(array_1d<double, 3>& rnormal){
 }
 
 
-double DEMWall::GetYoung()                                                      { return GetProperties()[YOUNG_MODULUS]; }
-double DEMWall::GetTgOfFrictionAngle()                                          { return GetProperties()[FRICTION]; }
-double DEMWall::GetPoisson()                                                    { return GetProperties()[POISSON_RATIO]; }
+double DEMWall::GetYoung()                                                      { return GetProperties()[YOUNG_MODULUS];              }
+double DEMWall::GetPoisson()                                                    { return GetProperties()[POISSON_RATIO];              }
+double DEMWall::GetTgOfFrictionAngle()                                          { return GetProperties()[FRICTION];                   }
+double DEMWall::GetTgOfStaticFrictionAngle()                                    { return GetProperties()[STATIC_FRICTION];            }
+double DEMWall::GetTgOfDynamicFrictionAngle()                                   { return GetProperties()[DYNAMIC_FRICTION];           }
```

Comment:
```suggestion
double DEMWall::GetTgOfDynamicFrictionAngle() const                                  { return GetProperties()[DYNAMIC_FRICTION];           }
```

If I may suggest, I recommend making this `const` for the following reasons:
- then you can also use it in other `const` functions
- More important, to make the access to the `Properties` threadsafe, see #6481

Reply:
Done! Thank you for the suggestion @philbucher